nemo_skills/training/data_preparation_utils/config/code_sft.yaml

```diff
@@ -24,11 +24,22 @@ assistant_end: null  # can be used to not use jinja and just add this suffix
 
 code_tags: null
 
+# input params
+min_input_length: 0
+max_input_length: 1024
```

Comment:
probably a good idea to not set this by default and require to be set by user. Both here and in math config. Other than that, lgtm

Reply:
Agreed. Set it to none / null by default

Reply:
Done.